whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/LinkRelationType.java

```diff
@@ -0,0 +1,32 @@
+package net.ripe.db.whois.api.rdap.domain;
+
+public enum LinkRelationType {
+    UP("rdap-up"),
+    TOP("rdap-top"),
+    DOWN("rdap-down"),
+    BOTTOM("rdap-bottom"),
+    ACTIVE("rdap-active");
+
+    private final String value;
+
+    LinkRelationType(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public static String concatActiveLinkRelation(final LinkRelationType type) {
```

Comment:
Is it more readable to make it generic such as `public static String concat(final LinkRelationType ... types)` ? Then you can specify `LinkRelationType.concat(UP, ACTIVE)` etc.

Reply:
Done